nodeadm/test/e2e/cases/kubelet-config-dir/config.kubelet-config-json.yaml

```diff
@@ -8,12 +8,11 @@ spec:
     certificateAuthority: Y2VydGlmaWNhdGVBdXRob3JpdHk=
     cidr: 10.100.0.0/16
   kubelet:
-    config: |
-      {
-        "kind": "KubeletConfiguration",
-        "apiVersion": "kubelet.config.k8s.io/v1beta1",
-        "clusterDNS": [
-          "0.0.0.0",
-          "1.1.1.1"
-        ]
-      }
+    config: {
```

Comment:
Is this an intended use case, embedding json inside the yaml? My impression was that users could supply the **entire** `NodeConfig` as either json or yaml, not a subset. I didn't realize this was even valid yaml to begin with but if the tests run and parse I guess it is.

Reply:
for drop-in use cases I can see users doing it for the `kubelet.config` even though converting it wouldn't take much effort

Reply:
All JSON is valid YAML, so I guess this is allowed per the specs. Looks pretty odd though 😄